pkgs/development/tools/misc/gdb/wrapper.nix

```diff
@@ -0,0 +1,32 @@
+{ stdenv, lib, makeWrapper, gdb-unwrapped, safePaths }:
+
+let
+  gdb = gdb-unwrapped;
+in
+  stdenv.mkDerivation {
+    name = gdb.name;
+    buildInputs = [ makeWrapper ];
+    propagatedBuildInputs = [ gdb ];
+    propagatedUserEnvPkgs = [ gdb ];
+    phases = "installPhase fixupPhase";
```

Comment:
Avoid setting phases. Instead, pass e.g. `dontBuild = true;`

Reply:
Ditto: using something like `buildCommand` instead doesn't make the man-page available.